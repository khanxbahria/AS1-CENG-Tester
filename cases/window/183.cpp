#include "../../your_code/Window.cpp"
#include "../../your_code/Browser.cpp"
#include "../../your_code/LinkedList.h"
int main() {
{
Window* wind = new Window();
Tab R = Tab("dd", "www","zz");
wind->changeActiveTabTo(2);
Tab K = Tab("uu", "yy","zz");
wind->changeActiveTabTo(3);
Tab J = Tab("k", "t","ccc");
wind->isEmpty();
Tab U = Tab("h", "rrr","zzz");
wind->closeTab();
Tab N = Tab("r", "vv","x");
wind->closeTab();
Tab C = Tab("ccc", "www","ww");
wind->moveActiveTabTo(1);
Tab V = Tab("bb", "s","ggg");
wind->print();
Tab M = Tab("zzz", "sss","aaa");
wind->isEmpty();
Tab O = Tab("bbb", "uu","ccc");
wind->changeActiveTabTo(1);
Tab H = Tab("q", "yy","ooo");
wind->print();
Tab B = Tab("dd", "ll","bbb");
wind->isEmpty();
Tab G = Tab("cc", "m","m");
wind->changeActiveTabTo(5);
Tab D = Tab("kk", "hh","f");
wind->print();
Tab Q = Tab("oo", "l","nn");
wind->isEmpty();
Tab T = Tab("xx", "zz","ooo");
wind->moveActiveTabTo(3);
Tab W = Tab("sss", "iii","qq");
wind->changeActiveTabTo(5);
wind->closeTab();
Tab L = Tab("q", "k","k");
wind->getActiveTab();
wind->newTab(B);
wind->closeTab();
wind->print();
Tab P = Tab("vv", "bbb","n");
wind->changeActiveTabTo(2);
wind->closeTab();
wind->newTab(C);
wind->isEmpty();
wind->print();
wind->isEmpty();
wind->print();
Tab A = Tab("s", "o","ccc");
wind->closeTab();
wind->getActiveTab();
wind->changeActiveTabTo(4);
wind->closeTab();
wind->getActiveTab();
wind->print();
Tab F = Tab("g", "sss","qqq");
wind->getActiveTab();
wind->moveActiveTabTo(1);
wind->newTab(A);
wind->isEmpty();
wind->closeTab();
Tab Z = Tab("p", "gg","ooo");
wind->newTab(D);
Tab E = Tab("uu", "qq","ww");
wind->moveActiveTabTo(3);
wind->print();
wind->isEmpty();
wind->isEmpty();
wind->newTab(E);
Tab X = Tab("d", "mmm","vvv");
wind->moveActiveTabTo(3);
wind->moveActiveTabTo(2);
wind->print();
Tab I = Tab("tt", "sss","ll");
wind->isEmpty();
wind->isEmpty();
wind->moveActiveTabTo(4);
wind->newTab(F);
wind->closeTab();
wind->isEmpty();
wind->closeTab();
wind->getActiveTab();
wind->moveActiveTabTo(1);
wind->moveActiveTabTo(1);
wind->newTab(G);
wind->changeActiveTabTo(2);
wind->isEmpty();
wind->changeActiveTabTo(2);
wind->getActiveTab();
wind->getActiveTab();
wind->getActiveTab();
wind->getActiveTab();
Tab S = Tab("qqq", "x","c");
wind->isEmpty();
wind->newTab(H);
wind->print();
wind->isEmpty();
wind->isEmpty();
wind->closeTab();
wind->moveActiveTabTo(4);
wind->getActiveTab();
wind->moveActiveTabTo(2);
wind->newTab(I);
wind->isEmpty();
wind->closeTab();
wind->closeTab();
wind->closeTab();
wind->moveActiveTabTo(0);
wind->newTab(J);
wind->closeTab();
wind->getActiveTab();
wind->newTab(K);
wind->moveActiveTabTo(2);
wind->getActiveTab();
wind->changeActiveTabTo(0);
wind->newTab(L);
wind->changeActiveTabTo(5);
wind->changeActiveTabTo(2);
wind->isEmpty();
wind->moveActiveTabTo(4);
wind->closeTab();
wind->getActiveTab();
wind->print();
wind->isEmpty();
wind->getActiveTab();
wind->newTab(M);
delete wind;
}
	return 0;
}