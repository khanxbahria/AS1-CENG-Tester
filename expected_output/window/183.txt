Window is empty
The list is empty.
Window is empty
The list is empty.
Window is empty
The list is empty.
Window is empty
The list is empty.
The active tab is tab 0
www
ccc
ww

The active tab is tab 0
www
ccc
ww

Window is empty
The list is empty.
The active tab is tab 0
hh
kk
f

The active tab is tab 1
hh
kk
f

qq
uu
ww

The active tab is tab 2
hh
kk
f

m
cc
m

yy
q
ooo

The active tab is tab 0
yy
uu
zz

